src/Games/NexusMods.Games.FOMOD/CoreDelegates/InstallerDelegates.cs

```diff
@@ -12,12 +13,19 @@ public sealed class InstallerDelegates : ICoreDelegates
     public IContextDelegates context { get; }
     public IIniDelegates ini => throw new NotImplementedException();
     public IPluginDelegates plugin { get; }
-    public IUIDelegates ui { get; }
 
-    public InstallerDelegates(ILoggerFactory loggerFactory, IGuidedInstaller guidedInstaller)
+    public IUIDelegates ui => UiDelegates;
+    public readonly UiDelegates UiDelegates;
```

Comment:
Why was this needed?

Reply:
The FOMOD installer needs to set a field inside the `UiDelegates` class.